docs/3.0rc/community/install-contribute.mdx

```diff
@@ -0,0 +1,285 @@
+---
+title: Install for development
+---
+
+Thanks for contributing to Prefect!
+
+## Set up a development environment
+
+First, download the source code and install an editable version of the Python package:
+
+```python
+# Clone the repository
+git clone https://github.com/PrefectHQ/prefect.git
+cd prefect
+
+# We recommend using a virtual environment
+
+python -m venv .venv
+source .venv/bin/activate
+
+# Install the package with development dependencies
+
+pip install -e ".[dev]"
+
+# Setup pre-commit hooks for required formatting
+
+pre-commit install
+
+```
+
+
+If you don't want to install the pre-commit hooks, manually install the formatting dependencies with:
+
+```
+pip install $(./scripts/precommit-versions.py)
+
+```
+
+
+Before a contribution is accepted, you need to run `ruff check` and `ruff format`.
+
+After installation, you can run the test suite with `pytest`:
+
+```
+# Run all the tests
+pytest tests
+
+# Run a subset of tests
+
+pytest tests/test_flows.py
+
+```
+
+<Tip>
+**Build the Prefect UI**
+
+If you intend to run a local Prefect server during development, first build the UI. 
+See [UI development](#ui-development) for instructions.
+</Tip>
+
+## Prefect Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making 
+participation in our project and our community a harassment-free experience for everyone. This is regardless of age, body size, 
+disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, 
+nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our standards
+
+Examples of behavior that contributes to creating a positive environment include:
```

Comment:
```suggestion
Examples of behavior that contribute to creating a positive environment include:
```

Reply:
I think "contributes" is matching "Examples of behavior", plural, as opposed to "behavior".

Reply:
Oh, you're totally right. What do you think of dropping "Examples of" such that it reads "Behavior that contributes.."

Reply:
Sounds good to me.